### PR TITLE
[Task] Stop unneeded circle services before executing

### PR DESCRIPTION
**Pull request by jbarnettwomply**

### CODE REVIEW

**Comment by nkuzmina** [comment]
merging...

File: circle/stop-competitors.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sudo service mysql stop
+sudo service postgresql stop
+sudo service mongodb stop
```
Comment (lfoos) [inline circle/stop-competitors.sh]: Where are we using mongo? @msilvey's head may explode....

Comment (jbarnettwomply) [inline circle/stop-competitors.sh]: yeah, we're  not using any of these things that circle "helpfully" starts up for us (even the services that we do use, we start our own docker'ed versions of)

Comment (lfoos) [inline circle/stop-competitors.sh]: IC, 👍

**Pull request merged**